src/bridge/mod.rs

```diff
@@ -139,11 +150,13 @@ impl Bridge {
         runtime.spawn(async move {
             start_process(receiver).await;
         });
-
-        Bridge { _runtime: runtime, sender }
+        Bridge { _runtime: runtime, sender, running: Arc::new(AtomicBool::new(true)) }
```

Comment:
being an atomic value, you can share references. No need to use an arc or clone the reference.